### PR TITLE
Add field kubernetes.volume.fs.inodes.pct

### LINKED ISSUE #30784: Add field `kubernetes.volume.fs.inodes.pct` to get used inodes percent
**Describe the enhancement:**

It would be great if there was an acutal value for the percentage of used inodes for a Kubernetes-Volume.
That would be a simple calculation with the existing data:

```
volumeEvent.Put("fs.inodes.pct", volume.InodesUsed/float64volume.Inodes)
```

Not sure about the best syntax though :
- kubernetes.volume.fs.inodes.pct
- kubernetes.volume.fs.used.pct

**Describe a specific use case for the enhancement or feature:**

Our Use-Case would be a simple montoring of the used inodes and an alert when a certain threshold is hit.

For example as soon as more than 90% of inodes is used, we want an alert so we can take the appropriate action (delete data, extend the volume etc.).

It would also make it way easier to create a used-inodes-visualization.

### PR DESCRIPTION
**Pull request by JulienOrain**
- Enhancement

## What does this PR do?

This PR adds kubernetes.volume.fs.inodes.pct field. 

This field is calculated as kubernetes.volume.fs.inodes.used / kubernetes.volume.fs.inodes.count

## Why is it important?

Provide used inodes insight with a direct metric.

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have made corresponding change to the default configuration files
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

## Author's Checklist

- [ ]

## How to test this PR locally

Run the testsuite

## Related issues

- Closes #30784

## Use cases

NA

## Screenshots

NA

## Logs

NA

### CODE REVIEW

**Review by tetianakravchenko — APPROVED** [review]

**Comment by tetianakravchenko** [comment]
Hi @JulienOrain, thank you for your contribution, could you please sign CLA?

**Comment by JulienOrain** [comment]
Hello @tetianakravchenko

Thanks for your review.

CLA is now signed.

Have a nice week ;-)

File: CHANGELOG-developer.next.asciidoc [hunk 1]
```diff
@@ -134,6 +134,7 @@ The list below covers the major changes between 7.0.0-rc2 and main only.
 - Add gcp firestore metricset. {pull}29918[29918]
 - Added TESTING_FILEBEAT_FILEPATTERN option for filebeat module pytests {pull}30103[30103]
 - Add gcp dataproc metricset. {pull}30008[30008]
+- Add kubernetes.volume.fs.inodes.pct field.
```
Comment (MichaelKatsoulis) [inline CHANGELOG-developer.next.asciidoc:137]:
```suggestion
- Add kubernetes.volume.fs.inodes.pct field. {pull}30785[30785]
```

**Review by MichaelKatsoulis — APPROVED** [review]
Thanks @JulienOrain . Just this small update!

**Comment by JulienOrain** [comment]
Hello @MichaelKatsoulis 
Thanks for your review.
I applied your suggestion.
Have a nice week !

**Comment by tetianakravchenko** [comment]
@JulienOrain could you please resolve conflict and check my comment above - https://github.com/elastic/beats/pull/30785#discussion_r827829983 ?

File: CHANGELOG-developer.next.asciidoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (JulienOrain) [reply CHANGELOG-developer.next.asciidoc]: done ;-)

**Comment by JulienOrain** [comment]
Cool, thanks for my first elastic PR ;-) !!

**Pull request merged**